crates/noirc_evaluator/src/ssa.rs

```diff
@@ -63,6 +63,8 @@ pub(crate) fn optimize_into_acir(
             // and this pass is missed, slice merging will fail inside of flattening.
             .mem2reg()
             .print(print_ssa_passes, "After Mem2Reg:")
+            .fold_constants()
+            .print(print_ssa_passes, "After Constant Folding:")
```

Comment:
Can you remove this?

Reply:
Sure, I had originally put it just to make sure that constant folding works before flattening.

Reply:
removed